src/ray/rpc/grpc_server.cc

```diff
@@ -58,8 +58,12 @@ void GrpcServer::Run() {
   // Build and start server.
   server_ = builder.BuildAndStart();
 
-  RAY_CHECK(server_) << "Failed to start the grpc server. The specified port is "
-                     << specified_port;
+  RAY_CHECK(server_)
+      << "Failed to start the grpc server. The specified port is " << specified_port
```

Comment:
what's the GRPC server?

Reply:
asking as a ray user

Reply:
Ah, good point. Let me try improving it.

Reply:
Check again?